pegasus/rake/seed.rake

```diff
@@ -166,14 +167,20 @@ class GSheetToCsv
       puts "Error on file: #{@gsheet_path}, #{e}"
       throw e
     end
-    if @exclude_columns.empty?
+    if @include_columns.empty? && @exclude_columns.empty?
```

Comment:
This seems fragile, better not to have it. (e.g. we could add new factors that influence the output other than @include_columns and @exclude_columns)